.github/workflows/statuses.yml

```diff
@@ -11,6 +11,7 @@ on:
 jobs:
   build-and-deploy:
     name: Update status project
+    if: ${{ github.repository == 'primer/react' }}
```

Comment:
Hi! Just checking, will this only run if the target for a pull request (from a fork) is primer/react and not otherwise?

Reply:
This workflow doesn't run on pull request, only push. But, yes, it will only run if the repository is `primer/react`.